.travis.yml

```diff
@@ -1,2 +1,10 @@
 language: ruby
+rvm:
+- 2.1
 script: "bundle exec jekyll build"
```

Comment:
I would change this to 
script:
 - bundle exec jekyll build

it just helps for the future.